joomla/joomla-cms
# Prefer HTTPS for the help screen proxy

The help screen proxy is another Joomla domain which is working fine with HTTPS and this PR updates references to that site.  Additionally, this will allow a site which is loading the admin via SSL to use the screens at `administrator/index.php?option=com_admin&view=help` (not possible presently if you have the HTTP URL entered due to mixed content issues).
### Testing Instructions

On an existing site, edit the `$helpurl` variable of your configuration.php file to use the HTTPS link.  For new installations, the `$helpurl` variable should default to HTTPS.
